File: iniciante/javascript/1117.js
const { count } = require('console')

var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')


let media = 0
let countNotas = 0

while(true){

    let nota = parseFloat(lines.shift().replace('\r', ''))
    if(nota >= 0 && nota <= 10){
        media += nota
        countNotas++
    }else{
        console.log('nota invalida')
    }
    if(countNotas == 2)
        break
}

console.log(`media = ${(media/countNotas).toFixed(2)}`)

File: iniciante/python/1117.py
# -*- coding: utf-8 -*-

count_notas_validas = 0
soma_notas = 0

while count_notas_validas < 2:
    entrada = input()
    if '.' in entrada:
        entrada = float(entrada)
    else:
        entrada = int(entrada)

    if entrada >= 0 and entrada <= 10:
        soma_notas += entrada
        count_notas_validas += 1
    else:
        print('nota invalida')

print(f'media = {(soma_notas / count_notas_validas):.2f}')
